src/utils/zooming.js

```diff
@@ -12,3 +12,60 @@ function wheelDelta(ev) {
 export function zoomFactor(ev) {
   return Math.exp(wheelDelta(ev));
 }
+
+// This hack was directly taken from https://github.com/facebook/react/issues/14856#issuecomment-478144231
+// It defaults document level events back to active. Defaulting them to passive was introduced in Chrome 73
+// (https://www.chromestatus.com/features/6662647093133312) and since then, our React onWheel event broke
+// timeline scrolling - see https://github.com/weaveworks/service-ui/issues/3750.
+// TODO: Remove this code once https://github.com/facebook/react/issues/14856 gets resolved.
+let madeReactEventsActive = false;
+if (!madeReactEventsActive) {
+  const EVENTS_TO_MODIFY = [
+    'touchstart',
+    'touchmove',
+    'touchend',
+    'touchcancel',
+    'wheel',
+  ];
+
+  const originalAddEventListener = document.addEventListener.bind();
+  document.addEventListener = (type, listener, options, wantsUntrusted) => {
```

Comment:
This is a bit scary replacing the global `addEventListener` fn.

Would an approach like we use in scope https://github.com/weaveworks/scope/blob/master/client/app/scripts/components/zoomable-canvas.js#L66 work too?

Reply:
Yup, I tried it and it seems to work - thanks for the tip!

PTAL